### DISCUSSION TITLE
Background music

### DISCUSSION THREAD

**Discussion by jordanh** [ParabolInc/parabol#5726] [Feedback]
## Issue - Enhancement

Opening this as a discussion

[A user](https://app.hubspot.com/contacts/3888472/contact/12204601) writes 🔒 :
> Music! purely a nice-to-have feature, but it would be great to have some way to play music while people are going through the "reflect" phase and working independently on the call.

### Acceptance Criteria (optional)
Users can:
   - A design issue that addresses the tension behind the above (and below as we collect it) feedback with AC describing a path forward

- **Estimated effort:** TBD

**Comment by jordanh** [comment]
See also: #3555 (Spotify integration)

Other audio related: #4773

**Comment by gcrickman** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/13321151) 🔒  says:

Maybe nice to have would to add some background music, to play when everybody is writing something

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/11903301) 🔒 says:

> When we are adding reflections, it would be great to have some background music.

**Comment by drewhousman1010** [comment]
A [user ](https://app.hubspot.com/contacts/3888472/contact/13165151)🔒 says: 

> I play music from my computer when my team is adding reflections so that there is no awkward silence

**Comment by mattkrick** [comment]
tagging @avivapinchas 
I had an interview today with a guy who made a minecraft skin customizer.
He said, "the minecraft music was the theme song to my life". Really made me wonder, maybe we should invest in original :notes: ?

**Comment by avivapinchas** [comment]
I think that's a great idea! @jordanh started this [brainstorm doc to figure out what type of mood.](https://www.notion.so/parabol/Background-Music-ebb2b67d419a4f2f9415d7a05362db42) 🔒

[a comment by an automated account is not shown]

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/41958351) 🔒  said: 

> I wish I could play some jeopardy music during our retros.

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/record/0-1/71812539258) 🔒  just told me on a call that they would love to have an option to play music during retros.

**Comment by drewhousman1010** [comment]
A [user ](https://app.hubspot.com/contacts/3888472/record/0-1/119467151) 🔒 said they want music during retros, especially when they are facilitating meetings for higher level VPs and not just their usual teammates. The awkward silences when you are with executives you don't know are even more awkward!

**Comment by charlesmathias** [comment]
A [user](https://parabol.slack.com/archives/C029ZTH0EL9/p1734450079687059) 🔒  said they want music in their retros during the reflect phase. Not necessarily a Spotify integration but some background music, "even elevator music"

**Comment by drewhousman1010** [comment]
A [longtime Team plan customer ](https://app.hubspot.com/contacts/3888472/record/0-1/127787851) 🔒 wrote into our support chat saying: 

> Subject: Background music in retro meetings
> Message: would be my fav feature - Background music in retro meetings
